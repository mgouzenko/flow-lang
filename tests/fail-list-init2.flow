// Should not be able to init list of differnt types

int main() {
  list <int> test = [1, 2, 3, 4, 5.0]; // FAIL
}