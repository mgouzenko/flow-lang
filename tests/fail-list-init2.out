Error in semantic analysis
List must be initialized with expressions of the same type
